holoviews/core/data/multipath.py

```diff
@@ -0,0 +1,160 @@
+import numpy as np
+
+from ..util import max_range
+from .interface import Interface
+
+class MultiInterface(Interface):
+    """
+    MultiInterface allows wrapping around a list of tabular datasets
+    including dataframes, the columnar dictionary format or 2D tabular
+    NumPy arrays. Using the split method the list of tabular data can
+    be split into individual datasets.
+    """
+
+    types = ()
+
+    datatype = 'multi'
+
+    subtypes = ['dataframe', 'dictionary', 'array', 'dask']
+
+    @classmethod
+    def init(cls, eltype, data, kdims, vdims):
+        new_data = []
+        dims = {'kdims': eltype.kdims, 'vdims': eltype.vdims}
+        extra_kws = {}
+        for d in data:
+            d, interface, dims, extra_kws = Interface.initialize(eltype, d, kdims, vdims,
+                                                                 datatype=cls.subtypes)
+            new_data.append(d)
+        return new_data, dims, extra_kws
+
+    @classmethod
+    def validate(cls, dataset):
+        pass
+
+    @classmethod
+    def template(cls, dataset):
+        """
+        Returns a Dataset template used as a wrapper around the data
+        contained within the multi-interface dataset.
+        """
```

Comment:
This seems to be an implementation detail specific to ``MultiInterface``. Do you see any reasons ``template`` should be exposed as part of the public interface API?

I also don't get why it is a template: the docstring suggests it could be called ``_metadataset`` or ``_multidataset``.